hack/lib/golang.sh

```diff
@@ -493,15 +496,16 @@ kube::golang::outfile_for_binary() {
 # Returns 0 if the binary can be built with coverage, 1 otherwise.
 # NB: this ignores whether coverage is globally enabled or not.
 kube::golang::is_instrumented_package() {
-  return $(kube::util::array_contains "$1" "${KUBE_COVERAGE_INSTRUMENTED_PACKAGES[@]}")
+  return "$(kube::util::array_contains "$1" "${KUBE_COVERAGE_INSTRUMENTED_PACKAGES[@]}")"
```

Comment:
this should not be a string

Reply:
@BenTheElder Thanks your comment. array_contains returns 0 or 1. So, this should not be a string. I will see your PR https://github.com/kubernetes/kubernetes/pull/75751